src/controller/product.controller.ts

```diff
@@ -1,32 +1,32 @@
-import express, { Request, Response } from "express";
-import productService from "../service/product.service";
+import { Request, Response } from 'express';
+import productService from '../service/product.service';
 
 export default {
   async getProductByName(req: Request, res: Response) {
     try {
-      const brand = req.params.brand;
+      const { brand } = req.params;
       const allBrandProduct = await productService.getProductByName(brand);
       if (allBrandProduct.length > 0) {
         res.status(200).send(allBrandProduct);
       } else {
         res.status(400).send("Brand doesn't exist in database");
```

Comment:
Does this need single quotes?

Reply:
Yes it does! I will fix this. Weird it didn't throw an error